src/plugins/condition/components/inspector/ConditionalStyle.vue

```diff
@@ -17,22 +17,12 @@ export default {
         'openmct'
     ],
     props: {
-        conditionIdentifier: {
-            type: Object,
+        conditionName: {
+            type: String,
             required: true
```

Comment:
If you are passing down conditionName (and making it required) you can remove the v-if. If you need the v-if control it should probably be bubbled up to the parent.

Reply:
This is temporary code that's going to change later. I won't touch this right now.